packages/manager/apps/key-management-service/src/data/hooks/useOKMS.ts

```diff
@@ -36,7 +36,6 @@ export const useOKMSById = (okmsId: string) => {
     ...{
       keepPreviousData: true,
```

Comment:
I know it has not been modified on this commit but `keepPreviousData` is deprecated with tanstackQuery v5 : [docs](https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#removed-keeppreviousdata-in-favor-of-placeholderdata-identity-function)